Gems/PhysX/Code/Source/EditorShapeColliderComponent.cpp

```diff
@@ -206,6 +222,12 @@ namespace PhysX
 
             break;
         }
+        case ShapeType::QuadSingleSided:
+        case ShapeType::QuadDoubleSided:
+        {
+            // a quad shape has no interior, just return an empty vector of sample points
+            break;
+        }
```

Comment:
uber nit: braces here are redundant, could just be 

```
case ShapeType::QuadDoubleSided:
    // a quad shape has no interior, just return an empty vector of sample points
    break;
```

Reply:
Removed the braces.